src/microbe_stage/editor/PatchMapEditorComponent.cs

```diff
@@ -132,6 +132,10 @@ public override void OnValidAction()
     {
     }
 
+    public override void OnLightLevelChanged(float lightLevel)
```

Comment:
I've got a feeling that there are more editor components that won't care about the light level so I think the base class should have a virtual empty method for this instead of needing to modify all of the components when minority of the components (I think) will even react to this.